### PR TITLE
Enhances challenge details view

### PR DESCRIPTION
**Pull request by mk4111**
See #664 

Updates styling, copy and tests

### CODE REVIEW

File: public/css/main.css [hunk 1]
```diff
@@ -604,17 +604,34 @@ Light-teal:     #66d9e0;
...
     .card__header {
         margin-bottom: 1rem;
         font-size: 1.125rem;
     }
         .card__header--challenge {
-            font-size: 1.125rem;
+            font-size: 1.1rem;
+            line-height: 1.3;
+        }
+        .card__header--editable {
```
Comment (mk4111) [inline public/css/main.css]: Will change this name to .card--editable so that it's applicable for everything, and not just headers

Comment (mk4111) [inline public/css/main.css]: Will make this .card__editable as width may need to be changed depending on view

File: templates/views/challenges/details.html [hunk 2]
```diff
@@ -9,31 +9,33 @@ <h1 class="title">Challenge</h1>
...
+      <a class="card__copy anchor anchor--shared-by" href="/orgs/{{challenge.org_id}}">
         {{challenge.org_name}}
       </a>
     </h4>
   </div>
   {{#if permissions.editable}}
-    <div class="card card--challenge card__suggested-matches">
-      <h2 class="card__data">Suggested Matches</h2>
+    <div class="card card__suggested-matches">
+      <h2 class="card__data card__header--suggested-matches">MATCHED WITH</h2>
```
Comment (harrygfox) [inline templates/views/challenges/details.html]: doesn't quite make semantic sense: is it a data element is it a modified header?
Whichever is the case then your elements and modifiers should marry up.
either go for a `.card__data--suggested-matches` modifier or make it a `card__header` element

Comment (mk4111) [inline templates/views/challenges/details.html]: Damn twu :squid: Thanks Hazza :1st_place_medal:

**Comment by harrygfox** [comment]
@mk4111 ready to be DIY merged once conflicts are resolved 👍

**Pull request merged**